lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/Builtin.java

```diff
@@ -160,6 +160,20 @@
      * type. Auto-registered methods do not have to be declared explicitly.
      */
     boolean autoRegister() default true;
+
+    /**
+     * Allow aggressive inlining or not. This argument doesn't need to be specified. If it is
```

Comment:
`inlineable` is a tri-state attribute. It can either be:
- missing - defaults are derived from arguments as described in the javadoc
- `true` - the builtin will share parent's `VirtualFrame`
- `false` - the builtin gets its own `VirtualFrame`

It is possible to make the `boolean` attribute tri-state as it is processed by annotation processor and the processor may check absence/presence/`true`/`false` states.